server/resources/[LOG]/webhook/server.lua

```diff
@@ -89,4 +89,26 @@ AddEventHandler("chatMessage", function(source, author, text)
   send_to_discord(what_were_sending, "chat")
   local for_splunk ={author = author, char =sourceCharacter.firstname .. " " .. sourceCharacter.lastname, message = text, }
   send_to_splunk(for_splunk, "chat")
+end)
+
+RegisterServerEvent("vorp:ImDead")
+AddEventHandler("vorp:ImDead", function(isDead)
+  local isDead = isDead 
+  local source = source
```

Comment:
same with here we should have source available, BUT we must accept source as param